neurofire/models/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .unet_multiscale.unet_3d_multiscale import UNet3DMultiscale
 
 from .fcn.fcn import FCN
+from .fcn.resnet3d import *
```

Comment:
Please don't use * imports, but just import the relevant classes / functions.
Otherwise we drag in torch and all the rest.

Reply:
Ok, nvm you have  a `__all__`